src/components/image-tooltip/ImageTooltipContent.tsx

```diff
@@ -3,34 +3,45 @@ import classNames from 'classnames';
 
 import './ImageTooltipContent.scss';
 
+type OnImageLoad = () => void;
+
 export type ImageTooltipContentProps = {
     /** A React component representing the image used in the visual tooltip */
     children: React.ReactElement;
     /** A string to be used in the tooltip's paragraph content */
     content: string;
+    /** A callback triggered onLoad for the image element */
+    onImageLoad?: OnImageLoad;
```

Comment:
should this be optional? seems it doesnt have to be (parent tooltip always passes it in i think?)

Reply:
I was thinking about `ImageTooltipContent` as a standalone component, but maybe it's always used via `ImageToolip` 🤔  I never know how abstract we want to be with open source-y stuffs.

Reply:
Since it lives within the image-tooltip folder it'd be okay to make it non-optional but doesnt really matter